discovery/dns/dns_test.go

```diff
@@ -23,9 +23,9 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/miekg/dns"
 	"github.com/prometheus/common/model"
-
 	"github.com/prometheus/prometheus/discovery/targetgroup"
 	"github.com/prometheus/prometheus/util/testutil"
+	"gopkg.in/yaml.v2"
```

Comment:
nit but it should be listed in this group:

```
	"github.com/go-kit/kit/log"
	"github.com/miekg/dns"
	"github.com/prometheus/common/model"
```

Reply:
the rule is

```
import (
  <stdlib packages>

  <3rd party packages>

  <github.com/prometheus/prometheus packages>
)
```

Reply:
ok